tracecat/tables/router.py

```diff
@@ -117,6 +118,182 @@ async def create_table(
             ) from e
 
 
+@router.post("/importer/get-types", status_code=status.HTTP_200_OK)
+async def get_column_types(
```

Comment:
confusing to have POST with a get, prefer to represent this as an action like "infer_column_types"